var/spack/repos/builtin/packages/py-enum34/package.py

```diff
@@ -29,15 +29,13 @@ class PyEnum34(PythonPackage):
     """Python 3.4 Enum backported to 3.3, 3.2, 3.1, 2.7, 2.6, 2.5, and 2.4."""
 
     homepage = "https://pypi.python.org/pypi/enum34"
-    url      = "https://pypi.python.org/packages/source/e/enum34/enum34-1.1.6.tar.gz"
```

Comment:
The desired default is that every package has a single `url` declared at the package level. Obviously, this isn't possible for all packages, as some versions have a single weird URL that cannot be extrapolated. For Python packages, we should be able to get everything from `pypi.io`, which makes life a lot easier. Otherwise, every time you add a new version, you have to add a `url` for that version, and it gets more complicated than necessary.